From 313840a8a8bcb7e3a4d1379177e04a287dc4a066 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Fri, 22 Mar 2019 02:25:45 +0000
Subject: [PATCH 146/151] i2c: altera: Enable Altera I2C driver for Stratix10

Add ARCH_STRATIX10 in the depends of Altera I2C driver so that
this I2C drvier is able to be used on Stratix10 platform.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 drivers/i2c/busses/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/i2c/busses/Kconfig b/drivers/i2c/busses/Kconfig
index ee5dfb5aee2a..f66921d674ee 100644
--- a/drivers/i2c/busses/Kconfig
+++ b/drivers/i2c/busses/Kconfig
@@ -364,7 +364,7 @@ comment "I2C system bus drivers (mostly embedded / system-on-chip)"
 
 config I2C_ALTERA
 	tristate "Altera Soft IP I2C"
-	depends on (ARCH_SOCFPGA || NIOS2) && OF
+	depends on (ARCH_SOCFPGA || ARCH_STRATIX10 || NIOS2) && OF
 	help
 	  If you say yes to this option, support will be included for the
 	  Altera Soft IP I2C interfaces on SoCFPGA and Nios2 architectures.
-- 
2.26.1

